app/assets/javascripts/discourse/app/components/sidebar/user/custom-sections.js

```diff
@@ -29,4 +43,10 @@ export default class SidebarUserCustomSections extends Component {
     });
     return this.currentUser.sidebarSections;
   }
+
+  _refresh() {
+    return ajax("/sidebar_sections.json", {}).then((json) => {
```

Comment:
See comment about just passing the sections in message bus instead of doing an AJAX call for everyone

Reply:
I would like to keep it simple because:
1. Probably next step will be allowing users to configure order of the sections, so calculation where to place a section might be a bit complicated;
2. I think that public section will be not changing very often. This is probably one time configuration option, so performance is not so crucial.